### PR TITLE
Errors #180 and #130

### PR DESCRIPTION
**Pull request by yangjackascd**
Unit testing in test file increased.
HTML code from models.py has been moved into their own separate HTML files.

### CODE REVIEW

**Comment by 13rac1** [comment]
Thanks for the pull request!
1. Please clean up these commits. They all can be squashed into one or two commits. Details: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html
2. This completely fixes #130 ? If, so it should say: "fixes #180" in the commit message preferably after the commit subject. Details: https://help.github.com/articles/closing-issues-via-commit-messages/
3. Does this fix #180? Either way, the commit message should say specifically what the new coverage level is.

Further comments inline.

File: roundware/rw/models.py [hunk 1]
```diff
@@ -16,6 +16,9 @@
 from roundwared.gpsmixer import distance_in_meters
 from django.contrib.auth.models import User
 from django.db.models.signals import post_save
+# Karl
```
Comment (13rac1) [inline roundware/rw/models.py:19]: Remove this name comment. Git blame will show who did what.

File: roundware/rw/models.py [hunk 2]
```diff
@@ -390,39 +393,59 @@ def media_display(self):
 
     def audio_player(self):
         if self.mediatype == 'audio':
-            return """<div data-filename="%s" class="media-display audio-file"></div>""" % self.filename
+            #return """<div data-filename="%s" class="media-display audio-file"></div>""" % self.filename
```
Comment (13rac1) [inline roundware/rw/models.py:396]: Remove all of these code removal comments. The git diff for this commit will show us what was removed if it needs to be replaced.

File: roundware/rw/templates/rw/model_audio_player.html [hunk 3]
```diff
@@ -0,0 +1 @@
+<div data-filename="%s" class="media-display audio-file"></div>
```
Comment (13rac1) [inline roundware/rw/templates/rw/model_audio_player.html:1]: File needs newline at end of file.

File: roundware/rw/templates/rw/model_text_display_div.html [hunk 4]
```diff
@@ -0,0 +1 @@
+</div>
```
Comment (13rac1) [inline roundware/rw/templates/rw/model_text_display_div.html:1]: I'm all for reducing code duplication, but does this single `</div>` need to be in a separate file?

File: tests/roundware/api1/test_commands.py [hunk 5]
```diff
@@ -19,7 +19,9 @@
                                      mock_distance_in_meters_far)
 from roundware.lib.exception import RoundException
 from roundware.api1.commands import (check_for_single_audiotrack, get_asset_info,
-                                     get_available_assets)
+                                     get_available_assets,get_parameter_from_request,
+op_log_event,play_asset_in_stream,get_config,get_tags_for_project,get_events,current_version)
```
Comment (13rac1) [inline tests/roundware/api1/test_commands.py:23]: Fix spacing. The o in `op_log_event` should be aligned under the g in `get_available_assets`

File: tests/roundware/api1/test_commands.py [hunk 6]
```diff
@@ -122,9 +124,16 @@ def test_get_asset_info(self):
                     "duraton_in_ms": 5}
         self.assertEquals(expected, get_asset_info(req))
 
+#Jake
     def test_play_asset_in_stream(self):
-        pass
+        req = FakeRequest()
```
Comment (13rac1) [inline tests/roundware/api1/test_commands.py:129]: These unit test changes should be a separate commit and pull request than the fixes for 130.

File: tests/roundware/api1/test_commands.py [hunk 7]
```diff
@@ -136,12 +145,39 @@ def test_vote_asset(self):
...
     def test_get_config(self):
-        pass
-
+        req = FakeRequest()
+        self.session = mommy.make(Session, project=self.project1,
+                                  language=self.spanish, id=2)
+        # need to test object speaker
+        req.GET = {'project_id':'1','latitude': '0.3', 'longitude': '0.3'}
+        getObject=get_config(req)
+        # in get_config, create_new_session != fales mean alway new Session
```
Comment (13rac1) [inline tests/roundware/api1/test_commands.py:156]: Spelling

File: tests/roundware/api1/test_commands.py [hunk 8]
```diff
@@ -695,6 +731,23 @@ def test_request_stream_just_one_speaker_in_range_needed(self):
     @patch.object(gpsmixer, 'distance_in_meters', mock_distance_in_meters_far)
     def test_project_out_of_range_message_localized(self):
         pass
+#Jake
```
Comment (13rac1) [inline tests/roundware/api1/test_commands.py:734]: Just mentioning it again. All `#name` needs to be removed. If Karl did something, he should be making his own commits.

File: tests/roundware/rw/test_models.py [hunk 9]
```diff
@@ -99,6 +110,81 @@ def setUp(self):
                                 asset=self.asset2, type="like")
         self.vote4 = mommy.make('rw.Vote', session=self.session1,
                                 asset=self.asset1, type="flag")
+    
+    #def test_clean_fields(self, exclude=None):
```
Comment (13rac1) [inline tests/roundware/rw/test_models.py:114]: Please don't add commented code.

**Comment by hburgund** [comment]
This may be useful for reference in the future, but is not going to be merged as is, so I am closing it.
